onelogin-aws-assume-role-cli/src/main/java/com/onelogin/aws/assume/role/cli/OneloginAWSCLI.java

```diff
@@ -24,149 +42,307 @@
+
+				if (i == 0) {
+					// AWS REGION
+					System.out.print("AWS Region (" + defaultAWSRegion + "): ");
+					awsRegion = scanner.next();
+					if (awsRegion.isEmpty() || awsRegion.equals("-")) {
+						awsRegion = defaultAWSRegion;
+					}
+				}
+
+				BasicAWSCredentials awsCredentials = new BasicAWSCredentials("", "");
+
+				AWSSecurityTokenServiceClientBuilder stsBuilder = AWSSecurityTokenServiceClientBuilder.standard();
+
+				AWSSecurityTokenService stsClient = stsBuilder.withRegion(awsRegion)
+						.withCredentials(new AWSStaticCredentialsProvider(awsCredentials)).build();
+
+				AssumeRoleWithSAMLResult assumeRoleWithSAMLResult = stsClient
+						.assumeRoleWithSAML(assumeRoleWithSAMLRequest);
+				Credentials stsCredentials = assumeRoleWithSAMLResult.getCredentials();
+				AssumedRoleUser assumedRoleUser = assumeRoleWithSAMLResult.getAssumedRoleUser();
+
+				if (profileName == null && file == null) {
+					String action = "export";
+					if (System.getProperty("os.name").toLowerCase().indexOf("win") != -1) {
+						action = "set";
+					}
+					System.out.println();
+					System.out.println("Assumed Role User: " + assumedRoleUser.getArn());
+					System.out.println("-----------------------------------------------------------------------");
+					System.out.println("| Success!                                                            |");
+					System.out.println("|                                                                     |");
+					System.out.println("| Temporary AWS Credentials Granted via OneLogin                      |");
+					System.out.println("|                                                                     |");
+					System.out.println("| Copy/Paste to set these as environment variables                    |");
+					System.out.println("-----------------------------------------------------------------------");
+					System.out.println();
+					System.out.println(action + " AWS_SESSION_TOKEN=" + stsCredentials.getSessionToken());
+					System.out.println();
+					System.out.println(action + " AWS_ACCESS_KEY_ID=" + stsCredentials.getAccessKeyId());
+					System.out.println();
+					System.out.println(action + " AWS_SECRET_ACCESS_KEY=" + stsCredentials.getSecretAccessKey());
+					System.out.println();
+				} else {
+					if (file == null) {
+						file = AwsProfileFileLocationProvider.DEFAULT_CREDENTIALS_LOCATION_PROVIDER.getLocation();
+					}
+					if (profileName == null) {
+						profileName = "default";
+					}
+					
+					
+					Map<String, String> properties = new HashMap<String, String>();
+					properties.put(ProfileKeyConstants.AWS_ACCESS_KEY_ID, stsCredentials.getAccessKeyId());
+					properties.put(ProfileKeyConstants.AWS_SECRET_ACCESS_KEY, stsCredentials.getSecretAccessKey());
+					properties.put(ProfileKeyConstants.AWS_SESSION_TOKEN, stsCredentials.getSessionToken());
+					properties.put(ProfileKeyConstants.ROLE_ARN, assumedRoleUser.getArn());
```

Comment:
Writing this line makes the `aws` not work for me. 

It returns ```Partial credentials found in assume-role, missing: 'source_profile'```

If you remove this line then the `aws` cli works as expected